up-l1/zenoh.adoc

```diff
@@ -138,33 +150,28 @@ UUri {
 }
 ----
 
-It's a local UUri, so it starts with `upl`.
-Also, we leave the second part blank since it doesn't have UAuthority.
-Only ID in UEntity and UResource matters while serializing into link:../basics/uri.adoc#421-local-micro-uri[micro-form Uri].
-Then transform it into lowercase hexadecimal representation.
-The Zenoh key is `upl/0100162e04d20100`.
+It's a local URI, and the short format of the UUri should be `/1/1234/5678` in this case.
+Let's assume users initialize the UPClienZenoh with the UAuthority `192.168.1.100`.
+Therefore, the Zenoh key will become `up/192.168.1.100/1/1234/5678`.
 
 * The remote UUri with UAuthority
 
 [source]
 ----
 UUri {
   authority: UAuthority {
-    name: UAuthName,
-    number: Id({01, 02, 03, 10, 11, 12})
+    name: "UAuthName",
```

Comment:
Please see https://github.com/eclipse-uprotocol/up-spec/pull/121, we are dropping id and ip from UAuthority.

Reply:
OK, I'll update it after the PR is merged.